datadog/api/metrics.py

```diff
@@ -57,3 +63,39 @@ def send(cls, *metrics, **single_metric):
             raise KeyError("'points' parameter is required")
 
         return super(Metric, cls).send(attach_host_name=True, **metrics_dict)
+
+    @classmethod
+    def query(cls, **params):
+        """
+        Query metrics from Datadog
+
+        :param start: query start timestamp
+        :type start: POSIX timestamp
+
+        :param end: query end timestamp
+        :type end: POSIX timestamp
+
+        :param query: metric query
+        :type query: string query
+
+        :return: JSON response from HTTP request
+
+        *start* and *end* should be less than 24 hours apart.
+        It is *not* meant to retrieve metric data in bulk.
+
+        >>> api.Metric.query(start=int(time.time()) - 3600, end=int(time.time()),
+                             query='avg:system.cpu.idle{*}')
+        """
+        # Set the right endpoint
+        cls._class_url = cls._METRIC_QUERY_ENDPOINT
+
+        # `from` is a reserved keyword in Python, therefore
+        # `api.Metric.query(from=...)` is not permited
+        # -> map `start` to `from` and `end` to `to`
```

Comment:
It's better anyway since the rest of the api seems to expect `start` and `end` parameters when searching for something (I checked `api//events.py`)